js/validateForm.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+function validate() {
+  const emailInputValue = document.getElementById('email').value;
+  const emailTest = /^([a-z\d-]+)@([a-z\d-]+)\.([a-z]{2,8})(\.[a-z]{2,8})?$/;
+
+  /* eslint-disable no-unused-vars */
+  document.forms[0].onsubmit = function testEmail(e) {
+    if (emailTest.test(emailInputValue)) {
+      document.getElementById('emailError').innerHTML = 'Expect amazing projects';
+      document.getElementById('emailError').style.visibility = 'visible';
+      document.getElementById('emailError').style.color = 'blue';
+    } else {
+      document.getElementById('emailError').innerHTML = 'Emails are case sensitive! please enter the email in lowercase';
+      document.getElementById('emailError').style.visibility = 'visible';
+      document.getElementById('emailError').style.color = 'red';
+      e.preventDefault();
+    }
```

Comment:
- [ ] According to the requirement, If a user submits the contact form with the email field containing only lowercase characters the form is sent normally. But I got a message when sending the form that contain the email in lowercase. please fix it.
- [ ] Please remove the `eslint-disable ` because it does not help to write clean code.
### Error
![Screenshot 2023-01-05 184720](https://user-images.githubusercontent.com/81727988/210786509-3ec6ce44-2367-4f46-931e-5f2a53780383.png)